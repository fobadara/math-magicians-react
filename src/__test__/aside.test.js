import { render, cleanup } from '@testing-library/react';
import Aside from '../components/aside';

afterEach(cleanup);

describe('App', () => {
  test('should create snapshot', () => {
    const { asFragment } = render(<Aside />);
    expect(asFragment(<Aside />)).toMatchSnapshot();
  });
});
