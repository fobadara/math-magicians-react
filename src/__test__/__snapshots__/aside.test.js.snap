// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should create snapshot 1`] = `
<DocumentFragment>
  <aside>
    Lets do some math!
  </aside>
</DocumentFragment>
`;
